testdata/gapi/onnx/download_onnx_models.py

```diff
@@ -0,0 +1,152 @@
+        os.replace(model_cache_model_path, self.MODELS_DIR + self.model_path)
+        # Rename cache file to old name
+        os.rename(model_cache_file_path, model_cache_model_path)
+        return self.verify()
+
+def download_cache():
+    print('______________________{}______________________'.format('Download cache'))
+    if not os.path.exists(CACHE_DIR):
+        print('[Info]: Cloning onnx_models repository from https://github.com/onnx/models.git')
+        os.system('git clone --recursive https://github.com/onnx/models.git onnx_models_cache')
+    else:
+        print('[Warn]: Directory already contains a "onnx_models_cache" folder - skipping')
+
+models = [
+    Model(
+        name ='ssd_mobilenet_v1_10',
+        filepath ='vision/object_detection_segmentation/ssd-mobilenetv1/model/',
+        sha = '6a2ba88990166b5212fc4115bb347dd1402fbf39'
+    ),
+    Model(
+        name='squeezenet1.0-9',
+        filepath='vision/classification/squeezenet/model/',
+        sha = '7c4a0cc990d877f46105eb331bb71e2c90c0ecbb'
+    ),
+    Model(
+        name='emotion-ferplus-8',
+        filepath='vision/body_analysis/emotion_ferplus/model/',
+        sha = '073ea68e09c0c8c21401b95e9cdccb42c639bc75'
+    ),
+    Model(
+        name='FasterRCNN-10',
+        filepath='vision/object_detection_segmentation/faster-rcnn/model/',
+        sha = '7df2f48a6429ea412733af8ce9673a092a5f84c4'
+    ),
+    Model(
+        name='yolov3-10',
+        filepath='vision/object_detection_segmentation/yolov3/model/',
+        sha = 'a3e31b46f37c2b5de0fc85b6b54571898e7bbbb7'
+    ),
+    Model(
+        name='tinyyolov2-8',
+        filepath='vision/object_detection_segmentation/tiny-yolov2/model/',
+        sha = '7ad8395edc8057030d17c14459de6d07f4d11ac6'
+    ),
+]
+
+if __name__ == '__main__':
+    selected_model_name = None
+    print_all_models = False
+    if len(sys.argv) > 1:
+        selected_model_name = sys.argv[1]
+        if (selected_model_name != 'print'):
+            print('Model: {}'.format(selected_model_name))
+        else:
+            print_all_models = True
+
+    download_cache()
+
+    failedModels = []
+    for m in models:
+        if (not print_all_models):
+            if selected_model_name is not None and not m.name.startswith(selected_model_name):
+                continue
+            if not m.download():
+                failedModels.append(m.model_path)
+        else:
+            print(m)
+    if failedModels:
+        print("Following models have not been downloaded:")
+        for f in failedModels:
+            print("* {}".format(f))
+
+    # "lfs" directory contains some data
+    # Remove this folder then "onnx_models_cache" will have standard size - 180 MB
+    # If we remove "onnx_models" folder than models will be downloaded again
+    # and "lfs" folder will exist during download
+    lfs_cache_path = CACHE_DIR + '.git/lfs'
+    if os.path.exists(lfs_cache_path):
+        rmdir_with_data(lfs_cache_path)
```

Comment:
@alalek, it is questionable step. Cache directory gets some data after calling `git lfs pull`. I remove this folder and don't know. Should cache directory have constant size (180 MB) or should contain received data (500 mb)? This (without deleting lfs folder) avoids downloading models after deleting `onnx_models` folder.